### ISSUE TITLE
nything typed after the initial "m" does not direct towards the messages app.

### ISSUE THREAD

**Issue by gse356** [briangonzalez/alfred-messages#24]
I'm a bit new to Alfreds and it did take a bit of tinkering to get yarn going butI'm pretty sure I got it to run properly and the warning about Alfred having no boundaries isn't something to worry about. See screenshot attached. 

![Screen Shot 2020-10-07 at 3 13 48 PM](https://user-images.githubusercontent.com/66913781/95376850-b726bb80-08af-11eb-956c-e02c552049d0.png)

but when I type "m steve hi Steve" I get the following:

![Screen Shot 2020-10-07 at 3 16 20 PM](https://user-images.githubusercontent.com/66913781/95377091-0f5dbd80-08b0-11eb-9828-74274ad4c572.png)
 
basically anything typed after the initial "m" does not direct towards the messages app.

Any insight would be appreciated.
